content/applications/sales/crm/acquire_leads/generate_from_website.rst

```diff
@@ -1,78 +1,106 @@
-available in Odoo Enterprise.
+To change the form to a specific sales channel, go to :menuselection:`Website --> Configuration -->
+Settings.` Under *Communication,** you will find where to change which *Sales Team* or
+*Salesperson* will be assigned to any incoming leads or opportunities from your Contact Form.
 
-.. image:: media/generate_from_website05.png
+.. image:: generate_from_website/contact-form-settings.png
    :align: center
+   :alt: Contact Form Settings
 
-Create a custom contact form
-----------------------------
+Creating a custom contact form
+==============================
 
-First, go to the page where you want to put your contact form. In edit mode, drag
-the form builder onto the page, and you will be able to add all the fields
-you wish.
+Wanting more info from visitors using the contact form? Contact forms can be customized for the
+specific information your team needs because they can generate multiple types of records in the
+system, such as emails, leads/opportunities, project tasks, helpdesk tickets, and more.
 
-.. image:: media/generate_from_website06.png
+The free *Form Builder* module can be installed from either the apps page or automatically when
+added to a page in the Website Builder. Adding a whole new form can be great if you are
+soliciting certain information that needs filtering, such as suggestions, bug reports, etc.
+
+.. image:: generate_from_website/form-building-blocks.png
+   :align: center
+   :alt: Form Builder building blocks
+
+However, if editing the available predefined options is all that’s necessary, editing the Contact
+Form itself is the best option.
+
+Editing contact form fields
+---------------------------
+In edit mode, click on the field to change, and you can edit any of the following information for
+the specific field:
+
+- **Type**: Choose a custom field option or an existing field. Examples include phone, file
+  upload, language, etc.
+- **Input Type**: This determines the type of entry customers should input. Available options are
+  text, email, telephone, and URL.
+- **Label Name**: Type in the display name to show users what information is needed from them.
+- **Label Position**: Choose the way the label is aligned with the rest of the form. The label
+  can be hidden, above the field, to the far left of the field, or right adjusted and closer to the
+  field.
+- **Required**: Toggleable option for information that you absolutely need entered
+- **Hidden**: Toggleable option to hide the given field for this form or not
+- **Shown on Mobile**: Toggleable option to show the field to users on mobile devices.
+
+.. image:: generate_from_website/editable-field-options.png
    :align: center
+   :alt: Editable field options
 
-By default, any new contact form will send an email. You can switch to
-lead/opportunity generation in *Change Form Parameters*.
+By default, any new contact form will send an email. You can change this to lead/opportunity
```

Comment:
```suggestion
A form's default action when a visitor completes it is to send you the completed information by email. You can change this to lead/opportunity
```
I tried rephrasing to make the content clearer. I'm not sure I succeeded though :)